UI_UX/Frontend/src/components/AppHeader.js

```diff
@@ -1,12 +1,29 @@
 import React from 'react';
-import logo from '../images/mybit-logo-white.png';
+import logo from '../images/mybit-logo-white.svg';
 import '../styles/AppHeader.css';
+import { Logo } from './Logo';
+import { ExchangeRate } from './ExchangeRate';
+import { AccountInfo } from './AccountInfo';
 
-export const AppHeader = () => {
+export const AppHeader = ({
+  loadingBalance = false,
+  exchangeRate = 2.23,
+  myBitBalance = 215,
+  loadingAccountInfo = false,
+  ethBalance = 20,
+  address = '0xde0BF ...'
+}) => {
   return (
     <div className="grid AppHeader">
-      <img className="AppHeader_logo" src={logo} />
-      <p className="AppHeader_connect">Connect account</p>
+      <Logo className="AppHeader_logo" />
+      <ExchangeRate value={exchangeRate} />
+      <AccountInfo
+        loadingBalance={loadingBalance}
+        myBitBalance={myBitBalance}
+        ethBalance={ethBalance}
+        loadingAccountInfo={loadingAccountInfo}
```

Comment:
If `loadingAccountInfo` is a `bool`, then use a proper variable name. Booleans use a confirmation or denial kind of variable, so do something like `isAccountInfoBeingLoaded` or `hasAccountInfo`.

This can be particularly nitpicky, but when you collaborate with other people, you need to make sure your variable names are clear anyone for anyone to understand.

Reply:
fixed